infra-l2-dynamo/template.yaml

```diff
@@ -115,6 +117,16 @@ Resources:
               - "clientSessionId"
               - "authSessionState"
             ProjectionType: "INCLUDE"
+        - IndexName: "authSessionState-index"
+          KeySchema:
+            - AttributeName: "authSessionState"
+              KeyType: "HASH"
+          Projection:
+            NonKeyAttributes:
+              - "sessionId"
+              - "createdDate"
+              - "reminderEmailSent"
```

Comment:
Does this index need to include `expiryDate` now that we're doing the expired check in the `getSessionsByAuthSessionState` function?

Reply:
Yeah will push soon just deploying against latest Dynamo stack to check we don't run into index issues

Reply:
Done